package.json

```diff
@@ -23,6 +23,7 @@
     "bootstrap": "5.1.3",
     "clsx": "^2.1.1",
     "date-fns": "^2.30.0",
+    "dompurify": "^3.1.7",
```

Comment:
I don't really see why we would need this.
The backend is already blocking < and > chars in the username field